src/data/get_flight_stats.py

```diff
@@ -27,18 +27,18 @@ class FlightStats:
     _flight_stats_df: pd.DataFrame = field(init=False)
 
     def __post_init__(self, flights: pd.DataFrame) -> None:
-        self._flight_stats_df = flights.map(self.count_flights)
+        flight_stats_list = list(map(self.count_flights, flights['flights'], flights['icao']))
         self._flight_stats_df = pd.DataFrame(
-            self._flight_stats_df['flights'].to_list(),
+            flight_stats_list,
             columns=['total_arrivals', 'total_departures'],
         )
 
     # total arrivals or departures
-    def count_flights(self, airport_flights: list[FlightDict]) -> list[int]:
+    def count_flights(self, airport_flights: list[FlightDict], icao: str) -> list[int]:
         departures = 0
         arrivals = 0
         if airport_flights:  # if the list is not empty
-            airport = airport_flights[0]['origin_iata']
+            airport = icao
```

Comment:
Why have we switched from `iata` to `icao`

Reply:
We didn't, big oops on my part